packages/rest/src/http-handler.ts

```diff
@@ -44,7 +44,7 @@ export class HttpHandler {
     return this._routes.describeApiPaths();
   }
 
-  findRoute(request: ParsedRequest) {
+  findRoute(request: ParsedRequest): ResolvedRoute {
```

Comment:
Note that the compiler was throwing me errors about `ResolvedRoute` being an unused import, yet, removing it would cause this function to fail compilation due to use of an unknown type.
(It's an intentional change, and an important one)